### PR TITLE
Added Button Shortcut Style Enhancements + Additional Storybook Tests

### PR DESCRIPTION
**Pull request by emshyu**
Resolves Issue #4871 (or enhances)

**Additions**
- **Customized Styling for Button Shortcuts:** Previously, button shortcuts/separator styling was standardized. We added customized styling for optional button shortcuts based on button accents and variants.
- **Enhanced Storybook Catalogs:** We modified ShortcutCatalog in Button.stories.tsx to display button shortcuts across styles and various inputs.

Screenshot 2024-12-07 at 9 16 35 PM
Screenshot 2024-12-07 at 9 16 30 PM
Screenshot 2024-12-07 at 9 16 22 PM

### CODE REVIEW

**Comment by FelixMalfait** [comment]
Screenshot 2024-12-08 at 18 42 24


Thanks @emshyu! I think the colors are different on Figma, also made a minor comment

https://twenty.com/developers/section/frontend-development/work-with-figma

**Comment by emshyu** [comment]
I just added in the admin token

File: package.json [hunk 1]
```diff
@@ -343,7 +343,7 @@
   },
   "license": "AGPL-3.0",
   "name": "twenty",
-  "packageManager": "yarn@4.4.0",
+  "packageManager": "yarn@4.5.2",
```
[the remark this answers is not in the document]
Comment (emshyu) [reply package.json:346]: Ah, I think it may have changed accidentally when I ran something with Yarn. This shouldn't be necessary.

**Comment by emshyu** [comment]
> Screenshot 2024-12-08 at 18 42 24
> Thanks @emshyu! I think the colors are different on Figma, also made a minor comment
> 
> https://twenty.com/developers/section/frontend-development/work-with-figma

Hi! I just updated the colors to match, should I make a new PR?

**Comment by emshyu** [comment]
> > Screenshot 2024-12-08 at 18 42 24
> > Thanks @emshyu! I think the colors are different on Figma, also made a minor comment
> > https://twenty.com/developers/section/frontend-development/work-with-figma
> 
> Hi! I just updated the colors to match, should I make a new PR?
Screenshot 2024-12-08 at 2 50 14 PM
Screenshot 2024-12-08 at 2 50 23 PM
Screenshot 2024-12-08 at 2 50 32 PM

**Review by FelixMalfait — APPROVED** [review]
Thank you!

**Pull request merged**